src/base/static/components/templates/map.js

```diff
@@ -99,6 +206,45 @@ class MapTemplate extends Component {
     });
   }
 
+  updateUIConfiguration(uiConfiguration) {
```

Comment:
All this logic used to live in the Backbone router. Previously we would set all this global state depending on the configuration of the map that a particular route needed before rendering the map template.